### PR TITLE
Custom items 12/01

### PR DESCRIPTION
**Pull request by Alberyk**
Adds the following items: 
https://forums.aurorastation.org/topic/13480-kasluul-solozels-maraziite-gear/
https://forums.aurorastation.org/topic/13426-suul-akhandis-staff/
https://forums.aurorastation.org/topic/13508-emily-cress-star-sweater-and-lyric-book/
https://forums.aurorastation.org/topic/13549-danny-harrows-laptop/
https://forums.aurorastation.org/topic/13396-anoxie-v804s-core/

### CODE REVIEW

File: code/modules/customitems/item_defines.dm [hunk 1]
```diff
@@ -3266,3 +3266,77 @@ All custom items with worn sprites must follow the contained sprite system: http
 	icon = 'icons/obj/custom_items/ioraks_cape.dmi'
 	icon_state = "ioraks_armband2"
 	item_state = "ioraks_armband2"
+
+
+/obj/item/clothing/accessory/poncho/fluff/vizili_mantle //Maraziite Throw Over - Kasluul Solozel - paradoxspace
```
Comment (ParadoxSpace) [inline code/modules/customitems/item_defines.dm]: wot

File: code/modules/customitems/item_defines.dm [hunk 2]
```diff
@@ -3266,3 +3266,77 @@ All custom items with worn sprites must follow the contained sprite system: http
...
+/obj/item/clothing/accessory/poncho/fluff/solozel_mantle //Maraziite Throw Over - Kasluul Solozel - paradoxspace
+	name = "maraziite throw over"
+	desc = "It's a grey poncho, exclusively donned by the members of the Maraziite Order. This one has a Izweski Hegemony flag boldly sewn onto the shoulder."
+	icon = 'icons/obj/custom_items/solozel_items.dmi'
+	icon_state = "solozel_mantle"
+	item_state = "solozel_mantle"
+	contained_sprite = TRUE
+	icon_override = FALSE
+
+/obj/item/clothing/mask/fluff/vizili_mask //Iron Mask - Kasluul Solozel - paradoxspace
```
Comment (ParadoxSpace) [inline code/modules/customitems/item_defines.dm]: also here

File: code/modules/customitems/item_defines.dm [hunk 3]
```diff
@@ -3266,3 +3266,77 @@ All custom items with worn sprites must follow the contained sprite system: http
...
+/obj/item/fluff/cress_book //Lyric Book - Emily Cress - mattatlas
+	name = "lyric book"
+	desc = "An old, faded folder containing various alphabetically organized lyrics of several songs, including musical sheets for guitars. A dark purple H is scribbled on the center, along with half a heart on the \
+	left and a cut on the bottom right. The lyrics inside have two copies each: one in Sol Common and one in Tau Ceti Basic. It generally looks to be hard rock."
+	icon = 'icons/obj/custom_items/cress_items.dmi'
+	icon_state = "cress_book"
+	w_class = 2
+	var/list/lyrics = list("Falling Down: A song about holding on to the last glimmer of hope. It's generally pretty motivational. The most recent song of the three.",
+							"Say Something New: A morose song about companionship, and being unable to continue without an undescribed dear friend. Morose, but overall motivational.",
+							"One By One -> A song telling an undescribed person to 'give it another try'. It seems to mostly about reconciliation and accepting failure. More somber than the others, and the most dated.")
```
Comment (NonQueueingMatt) [inline code/modules/customitems/item_defines.dm]:
```suggestion
							"One By One: A song telling an undescribed person to 'give it another try'. It seems to mostly about reconciliation and accepting failure. More somber than the others, and the most dated.")
```

File: code/modules/customitems/item_defines.dm [hunk 4]
```diff
@@ -3266,3 +3266,77 @@ All custom items with worn sprites must follow the contained sprite system: http
...
+							"One By One: A song telling an undescribed person to 'give it another try'. It seems to mostly about reconciliation and accepting failure. More somber than the others, and the most dated.")
+
+/obj/item/fluff/cress_book/attack_self(mob/user)
+	user.setClickCooldown(DEFAULT_ATTACK_COOLDOWN)
+	user.visible_message("<span class='notice'>[user] begins searching through \the [src] pages.</span>")
+	if(do_after(user, 25))
+		to_chat(user, "<span class='notice'>You read on the pages of \the [src]: [pick(lyrics)]</span>")
+
+
+/obj/item/modular_computer/laptop/preset/fluff/harrow_laptop //Developer's Laptop - Danny Harrow - brainos
```
Comment (Arrow768) [inline code/modules/customitems/item_defines.dm]: Would imho be better to `/obj/item/modular_computer/laptop/fluff/harrow_laptop` and add add the `install_default_hardware` proc to `/fluff` to maintain better future compatibility (so its not changed by things that effect the "normal" presets).

**Pull request merged**